scout/server/links.py

```diff
@@ -538,6 +547,32 @@ def gnomad_link(variant_obj, build=37):
     return url_template
 
 
+def gnomad_sv_link(variant_obj, build=37):
+    """Compose link to gnomAD website for a structural variant.
+
+    GnomAD SVs are not yet public for hg38 or MT as 2023-02-01.
+
+    Since we do not track the GnomAD variant ID we link to the region view instead.
+    For balanced variants, we pick the region for the start position (the var is available in both).
+    """
+
+    if build == 38 or variant_obj["chromosome"] in ["M", "MT"]:
```

Comment:
why not returning none?

Reply:
I was thinking that page explains why there is not connection, but either works!

Reply:
I'm thinking that perhaps there won't be any gnomAD frequency if the relative page doesn't exist in gnomAD